### ISSUE TITLE
"invalid filter method (73)" error on decoding a valid APNG image

### ISSUE THREAD

**Issue by Shnatsel** [image-rs/image#1237]
This happens in `image` from git, revision 092db1e21b3e8736711e2cbe1d9b5245613756c1

## Expected

Image decodes successfully

## Actual behaviour

thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Decoding(DecodingError { format: Exact(Png), underlying: Some(Format("invalid filter method (73)")) })', src/main.rs:3:18

## Reproduction steps

The reproducing sample could not be posted on github, I've supplied it privately to the maintainers. The sha256 of it is 11b15a255ade1e043e0e416b00bcd6b38f04133bae02071bf2a12b3114408587

```rust
fn main() -> std::io::Result<()> {
    let path = std::env::args().skip(1).next().unwrap();
    let image = image::io::Reader::open(path)?.guess_format().decode().unwrap();
    Ok(())
}
```

**Comment by HeroicKatora** [comment]
This one is fixed by [a completed PR on png](https://github.com/image-rs/image-png/pull/205). Closing once a release with that PR has been made.

**Comment by HeroicKatora** [comment]
I've verified by building `emulsion` with the fixed branch. It looks like it properly decodes all of the small and large details of the image.

**Comment by HeroicKatora** [comment]
Fixed by updating `png` to `0.16.4`.
